### PR TITLE
Add GCI logo

### PR DESCRIPTION
**Pull request by tioirawan**
This will add Google Code In logo and link to GCI official pages, this is also add Open Graph title, desc, and image

Closes https://github.com/coala/gci-leaders/issues/25

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -32,6 +32,13 @@ ul {
   padding-left: 1em;
 }
 
+.gci-logo{
```
Comment (jayvdb) [inline static/css/main.css]: Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*

Comment (blazeu) [reply static/css/main.css]: Also add space before {

File: templates/main.html [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
     <title>GCI Leaders</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta property="og:title" content="GCI Current Leaders">
+    <meta property="og:description" content="Google Code-in 2017 Current Leaders">
+    <meta property="og:image" content="{{rootURL}}/static/images/GCI-logo.jpg">
     <link rel="stylesheet" href="static/css/main.css">
   </head>
   <body>
-    <h1>GCI Current Leaders</h1>
+    <div class="header">
```
Comment (blazeu) [inline templates/main.html]: Unused class

**Review by blazeu — CHANGES_REQUESTED** [review]

**Comment by blazeu** [comment]
Hmm, do we really need to put the logo in our repo?
Does `og:image` needs to be in the same host?

**Comment by tioirawan** [comment]
no, but I cropped this logo from original image https://developers.google.com/open-source/gci/resources/downloads/GCI-new-logo.jpg

**Review by blazeu — CHANGES_REQUESTED** [review]
Format your commit message according to https://coala.io/commit

>Commit Body
>72 chars excluding newline for each line.

Also your commit title could be better.

**Review by blazeu — CHANGES_REQUESTED** [review]
Put the attribution in the footer.

https://developers.google.com/open-source/gci/resources/media

>Google Code-in and the Google Code-in logo are trademarks of Google Inc.

File: templates/main.html [hunk 3]
```diff
@@ -42,6 +50,9 @@ <h3>
         </div>
       {{/orgs}}
     </div>
+    <footer>
+      <small>Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small>
+    </footer>
```
Comment (blazeu) [inline templates/main.html:55]: It'll look better if the text is centered

**Comment by blazeu** [comment]
Omit "will" from the commit body.

```
This adds
```

File: templates/main.html [hunk 4]
```diff
@@ -42,6 +50,11 @@ <h3>
         </div>
       {{/orgs}}
     </div>
+    <footer>
+      <center>
+        <small>Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small>
+      </center>
```
Comment (blazeu) [inline templates/main.html]: Use css.

`<center>` is deprecated.

File: generate.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ const ncp = require('ncp').ncp
 const orgs = require('./out/data.json')
 
 const datetime = new Date(fs.statSync('./out/data.json').mtime).toUTCString()
+const rootURL = process.env.DEPLOY_PRIME_URL || process.env.URL
```
Comment (blazeu) [inline generate.js]: `DEPLOY_PRIME_URL` always contains the branch name :/ e.g. `http://master--gci-leaders.netlify.com`.
That is not ideal.

Use only `URL`.

**Review by blazeu — CHANGES_REQUESTED** [review]

File: templates/main.html [hunk 6]
```diff
@@ -4,10 +4,18 @@
     <meta charset="utf-8">
     <title>GCI Leaders</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta property="og:title" content="GCI Current Leaders">
+    <meta property="og:description" content="Google Code-in 2017 Current Leaders">
+    <meta property="og:image" content="{{rootURL}}/static/images/GCI-logo.jpg">
```
Comment (blazeu) [inline templates/main.html]: The value is being escaped by Mustache

Use triple bracket `{{{rootURL}}}` to prevent that.

**Review by blazeu — CHANGES_REQUESTED** [review]

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/41/commits/45e8aef8f9fdbef04fa2a83940701290a695b76b

**Comment by blazeu** [comment]
Submit your task :tada:

**Comment by tioirawan** [comment]
Done :)

**Pull request merged**